dom.bs

```diff
@@ -1098,6 +1100,26 @@ constructor steps are to do nothing.
 if this would be useful for your programs. For now, all author-created {{EventTarget}}s do not
 participate in a tree structure.</p>
 
+<p>The <dfn>default passive value</dfn>, given an event type |type| and an {{EventTarget}}
+|eventTarget|, is determined as follows:
+
+<ol>
+ <li>
+  <p>Return true if all of the following are true:
+
+  <ul>
+   <li><p>|type| is one of "<code>touchstart</code>", "<code>touchmove</code>",
+   "<code>wheel</code>", or "<code>mousewheel</code>" [[TOUCH-EVENTS]] [[UIEVENTS]]
+
+   <li><p>|eventTarget| is a {{Window}}, or is a {{Node}} whose <a>node document</a> is
```

Comment:
Nits:
* Window object*
* `<a for=/>node</a>` instead of {{Node}} (multiple times).